src/ophys_etl/modules/segmentation/qc/seed.py

```diff
@@ -0,0 +1,75 @@
+import h5py
+import numpy as np
+from pathlib import Path
+from typing import Union, Optional
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+
+def add_seeds_to_axes(
+        figure: Figure,
+        axes: Axes,
+        seed_h5_group: Union[str, h5py.Group] = "seeding",
+        image_background: Optional[Union[np.ndarray, str]] = "seed_image",
+        seed_h5_path: Optional[Union[str, Path]] = None):
+    """plots seeds optionally on top of the seeding image
+
+    Parameters
+    ----------
+    axes: matplotlib.axes.Axes
+        the axes to which to add the seeds
```

Comment:
Might be super obvious, but wouldn't hurt to mention that this function will mutate the input `Axes`.

Reply:
given the name of the method, and the docstring, I think it is super-obvious